### ISSUE TITLE
Is purchase_title officially supported?

### ISSUE THREAD

**Issue by alexweinstein** [soundcloud/api#237]
I'd like users to be able to update the Buy URL as well as the Buy Title for each track. As I'm working on the functionality, it seems that the option to update a track's Buy Title is limited to certain accounts (possibly paid ones). Even on SoundCloud.com. 

**SoundCloud Next User** - Updating does not work for this users
- PUT `purchase_url` does not cause an error
- PUT `purchase_title` does not cause an error
- GET `users/{id}/tracks/{track_id}` return contains updated `purchase_url`, but **null** `purchase_title`

**SoundCloud Next Pro User** - Updating works for this users
- PUT `purchase_url` does not cause an error
- PUT `purchase_title` does not cause an error
- GET `users{id}tracks/{track_id}` return contains updated `purchase_title` and `purchase_url`

This also seems to be the case on SoundCloud.com metadata editor. See screenshots below. 

Is `purchase_title` limited to paid accounts? Is this a bug? Or is the `purchase_title` a field that is not officially supported and should be avoided?

<img width="867" alt="Image 2023-02-03 at 8 50 48 AM" src="https://user-images.githubusercontent.com/222681/216620807-1184b496-20b2-40a1-89af-f83b7b323929.png">

<img width="866" alt="Image 2023-02-03 at 8 49 14 AM" src="https://user-images.githubusercontent.com/222681/216620824-71271ec1-86bd-4d43-a052-6fe1d668035e.png">

@rahul-sc

**Comment by youssefhassan** [comment]
Hey @alexweinstein, does this issue still persist?

**Comment by alexweinstein** [comment]
> Hey @alexweinstein, does this issue still persist?

Yup. This is still the behavior. Unsure if this is a bug or just a limit on paid accounts.
